### PR TITLE
helm: Support add deployment labels for istio-operator

### PR DESCRIPTION
**Pull request by my-git9**
**Please provide a description of this PR:**
helm: Support add deployment labels for istio-operator

### CODE REVIEW

**Review by zirain — CHANGES_REQUESTED** [review]
IMO, this's meaningless to an operator chart.

File: manifests/charts/istio-operator/templates/deployment.yaml [hunk 1]
```diff
@@ -3,6 +3,13 @@ kind: Deployment
 metadata:
   namespace: {{.Release.Namespace}}
   name: istio-operator{{- if not (eq .Values.revision "") }}-{{ .Values.revision }}{{- end }}
+  labels:
+    istio.io/rev: {{ .Values.revision | default "default" }}
+    operator.istio.io/component: "IstioOperator"
+    release: {{ .Release.Name }}
+    {{- range $key, $val := .Values.Labels }}
+    {{ $key }}: "{{ $val }}"
```
Comment (ymesika) [inline manifests/charts/istio-operator/templates/deployment.yaml:11]: I know it has been copied from the template labels but I think the quotation marks are going to be set (if necessary) in the values file so no need to add them again here.
wdyt?

Comment (my-git9) [reply manifests/charts/istio-operator/templates/deployment.yaml]: Updated

**Pull request closed without merging**